src/list/manage/containers/connected-promote-banner.js

```diff
@@ -9,27 +9,32 @@ import React from "react";
 import { connect } from "react-redux";
 
 import { PromoteDeviceBanner, PromoteFxABanner } from "../components/promote-banner";
-import { openHomepage, openSyncPrefs } from "../../actions";
+import { openHomepage, openSyncPrefs, hidePromo } from "../../actions";
 
-export function CurrentPromotionBanner({hasProfile, onClickDeviceBanner, onClickFxABanner}) {
+export function CurrentPromotionBanner({hasProfile, showPromo, onClickDeviceBanner, onClickFxABanner, onClose}) {
+  if (!showPromo) { return null; }
```

Comment:
Now that I look at this, it probably makes more sense for the parent component to just not render the banner if showPromo is false.